core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/statements/StatementsController.java

```diff
@@ -246,13 +248,24 @@ private ModelAndView getSparqlUpdateResult(Repository repository, HttpServletReq
 			}
 		}
 
-		try {
+		final String timeout = request.getParameter(Protocol.TIMEOUT_PARAM_NAME);
```

Comment:
This is the same code as the one in the RepositoryController. We should probably create a method in Protocol that contains the logic for parsing that parameter?